pages.fr/linux/pacman.md

```diff
@@ -0,0 +1,35 @@
+# pacman
+
+> Outil de gestion de paquets sur Arch Linux. 
+
+- Synchronise et mets à jour tous les paquets:
```

Comment:
I remember that in French you should use a space before `:`:
```suggestion
- Synchronise et mets à jour tous les paquets :
```
Please update all the example descriptions.

Reply:
Yep, I added them.